src/theory/quantifiers/bv_inverter.cpp

```diff
@@ -53,20 +53,8 @@ Node BvInverter::getSolveVariable(TypeNode tn)
   if (its == d_solve_var.end())
   {
     std::stringstream ss;
-    if (tn.isFunction())
-    {
-      Assert(tn.getNumChildren() == 2);
-      Assert(tn[0].isBoolean());
-      ss << "slv_f";
-    }
-    else
-    {
-      ss << "slv";
-    }
+    ss << "slv";
     Node k = NodeManager::currentNM()->mkSkolem(ss.str(), tn);
```

Comment:
You can pass "slv" directly to `mkSkolem()`, then `ss` is not needed.

Reply:
fixed